src/symbol/shaping.js

```diff
@@ -199,7 +200,8 @@ function shapeText(text: Formatted,
         left: translate[0],
         right: translate[0],
         writingMode,
-        lineCount: lines.length
+        lineCount: lines.length,
+        yOffset: -17 // the y offset *should* be part of the font metadata
```

Comment:
This is confusing to me. If the comment is suggesting that the `yOffset` is hard-coded here because it is not currently part of the font metadata, can that be clarified a bit. Is that going to be worked on in a separate ticket?

Reply:
It's confusing to me too. This comment predates this PR, I just moved it - it looks like it was introduced [here](https://github.com/mapbox/mapbox-gl-js/commit/52745740dfcd6d338959716797cfd6160ad1fea9#diff-e839bb19d4babbd27731bbde28317fc1R84), perhaps @ansis can shed some light on the matter?